src/SeExpr2/Vec.h

```diff
@@ -22,6 +22,13 @@
 #include <iostream>
 #include "Platform.h"
 
+// To fix differences in template TYPENAME resolution between MSVC and other compilers
+#if defined(WINDOWS)
+#   define TYPENAME
+#else
+#   define TYPENAME typename
+#endif
+
```

Comment:
The `typename` differences are unfortunate.  Is this something that'll be fixed in a future version of MSVC?  I can only hope ;-)

Thanks for these updates nonetheless!

Reply:
MSVC team has been putting a lot of work into their compiler and STL these last year(s), but I guess it'll take a bit more work/time to reach full conformance :p